dashboard/app/views/admin_reports/level_answers.csv.erb

```diff
@@ -6,5 +6,5 @@ CSV.generate do |csv|
       csv << response
     end
   end
-end
+end .html_safe
```

Comment:
Is this safe? Note that some of the answers will be arbitrary text entered by the user. Is it sanitized (escaped) before entry into the DB?